context/app/static/js/components/Contexts.jsx

```diff
@@ -0,0 +1,8 @@
+import React, { useContext } from 'react';
+
+const FlaskDataContext = React.createContext({});
+
+FlaskDataContext.displayName = 'FlaskDataContext';
+
+export { FlaskDataContext };
+export const useFlaskDataContext = () => useContext(FlaskDataContext);
```

Comment:
Prefer to define `useFlaskDataContext` below `FlaskDataContext` and export them in the same statement.